Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -531,6 +528,12 @@ class BrowserViewController: UIViewController {
                 profile.prefs.setString(appVersion, forKey: "latestAppVersion")
             }
         }
+
+        if var queuedAlertInfo = tabManager.selectedTab?.dequeueJavascriptAlertPrompt() {
+            let alertController = queuedAlertInfo.alertController()
+            alertController.delegate = self
```

Comment:
Ah, OK. If you decide to land the mini follow-up that puts this in the helper, would you mind also adding a comment here about this being an alternative to using `didSelectedTabChange`?